website/source/docs/builders/azure.html.md

```diff
@@ -57,7 +57,7 @@ builder.
 
 ### Optional:
 
--   `cloud_environment_name` (string) One of `Public`, `China`, or
+-   `cloud_environment_name` (string) One of `Public`, `China`, `German`, or
```

Comment:
You are right.  This is the name used in go-autorest, and it's also in the XPlat CLI.  Clearly there is a confusion between people and country.  

Would you care to take up the torch on this one? :grinning: